hazelcast/src/main/java/com/hazelcast/util/HashUtil.java

```diff
@@ -494,7 +494,7 @@ public static int hashToIndex(int hash, int length) {
      * keys are nearly-ordered by their hashed values so when adding one container's
      * values to the other, the number of collisions can skyrocket into the worst case
      * possible.
-     * <p/>
+     * <p/>f
```

Comment:
I think this doesn't belong here. I would recommend a direct edit after merge, if there is no other need for an PR update.